src/main/java/net/openhft/posix/internal/PosixAPIHolder.java

```diff
@@ -16,9 +17,10 @@ public class PosixAPIHolder {
                     ? new JNRPosixAPI()
                     : new WinJNRPosixAPI();
         } catch (Throwable t) {
-            LoggerFactory.getLogger(PosixAPIHolder.class).debug("Unable to load JNRPosixAPI", t);
-
+            LoggerFactory.getLogger(PosixAPIHolder.class).warn("Unable to load JNRPosixAPI", t);
+            posixAPI = new NoOpPosixAPI();
```

Comment:
I suspect we should abort here and/or fail fast
Anything which genuinely depends on Posix will likely break with a NoOp implementation.

What's motivating the change?

Reply:
This is @peter-lawrey request that we have an empty impl.

The motivation is [that we have seen ](https://github.com/ChronicleEnterprise/Chronicle-FIX-Demo/issues/326) and similar

Reply:
The key use here is `msync`, and not having it available on say Windows or Mac OSX shouldn't cause the queue to fail to start. Perhaps certain operations should fail outright, rather than always returns an error, but this should be on a case-by-case basis.